types/chrome/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     },
     "files": [
         "index.d.ts",
-        "test/index.ts"
+        "test/index.ts",
+	    "test/chrome-cast.ts"
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/chrome/tsconfig.json;
  Line: 23;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.